### PR TITLE
MC-40589: Document page builder preview limitation for filtering/sorting configurable, grouped and bundle products by price.

### PR DESCRIPTION
**Pull request by bubasuma**
## Purpose of this pull request

This pull request (PR) adds a warning information about page builder preview limitation for filtering/sorting configurable, grouped and bundle products by price.

## Magento release version

- [x] 2.4.x

- [x] 2.3.x

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [x] Other feature set, please specify: Page Builder extension

## Affected documentation pages

- https://docs.magento.com/user-guide/cms/page-builder-add-content-products.html

### CODE REVIEW

File: src/cms/page-builder-add-content-products.md [hunk 1]
```diff
@@ -134,6 +134,9 @@ Complete the _Settings_ according to the following sections:
    {:.bs-callout-info}
    Some products in the Category, SKU, or Condition settings may not appear in your products grid or carousel. For example, disabled products, products marked as not visible, out-of-stock products, and products assigned to another website will not be displayed.
 
+   {:.bs-callout-warning}
+   Configurable, grouped and bundle (dynamic price) products price is undefined in admin area. Therefore, such products will not be displayed in the **preview** if the products are filtered by price. Also, such products will not be ordered correctly in the **preview**, if the products are ordered by price.
```
Comment (dobooth) [inline src/cms/page-builder-add-content-products.md:138]:
```suggestion
   Prices for configurable, grouped and bundled (dynamic price) products are undefined in the Admin area. Therefore, these products are not displayed in the **preview** if the products are filtered by price. These products cannot be ordered correctly in the **preview** if ordered by price.
```

**Review by dobooth — CHANGES_REQUESTED** [review]

**Comment by dobooth** [comment]
Please set this so we can update this PR: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork

**Comment by bubasuma** [comment]
Hi @dobooth, I think in order for it to work you need write access to our L3 organization merchdocs fork, I can’t grant it, but you can request it in #github-permissions Slack channel.

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
running tests

**Pull request merged**